pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -88,6 +88,11 @@ const (
 	MemoryHotplugBlockAlignmentBytes = 2097152
 )
 
+const (
+	graphicsDeviceDefaultHeads uint = 1
+	graphicsDeviceDefaultVRAM  uint = 16384
+)
+
```

Comment:
just an idea:
Maybe we can move this inside `archconverter.go` file

Reply:
I have moved it. It can be argued that it makes sense in both files, but in the end the archconverter is using it.